project/src/main/world/environment/pebble-overworld-tiler.gd

```diff
@@ -2,23 +2,23 @@ tool
 extends Node
 ## Adds pebble tiles to an overworld terrain map.
 ##
-## Pebble tiles are added to a random selection of frosted/unfrosted cake tiles.
+## Pebble tiles are added to a random selection of gooped/ungooped cake tiles.
```

Comment:
Instead of 'gooped' and 'ungooped', terms like 'goopy' and 'goopless' are closer to english words